client/connector_create.go

```diff
@@ -102,7 +107,10 @@ func (cli *VanClient) ConnectorCreateFromFile(ctx context.Context, secretFile st
 	for _, oldSecret := range secrets.Items {
 		oldConnectionAuthor, ok := oldSecret.Annotations["skupper.io/generated-by"]
 		if !ok {
-			return nil, fmt.Errorf("A secret has no author.")
+			if !alreadyWarned {
```

Comment:
Lean towards not generating a log:

1. The client pattern is to return error indication and leave it to the caller to generate an indication or prompt to the user
2. With the 0.3 release, a user can make multiple connections "silently"